fuzzers/030-iob/bits.dbf

```diff
@@ -1 +1,3 @@
 38_98 39_97 39_99,LIOB33.IOB_Y1.PULLTYPE.PULLDOWN
```

Comment:
What happened to IOB_Y0?

Reply:
I think IOB_Y1 is used for inputs whereas IOB_Y0 for outputs. Drive and slew are properties for outputs only. Do you think we should have a separate entry in the dbf file for IOB_Y0 that is exactly as for IOB_Y1?

Reply:
That is incorrect.  IOB_Y1 and IOB_Y0 can be both inputs and outputs.

Reply:
I state corrected, I rebased the PR and now we have IOB_Y0 as well as IOB_Y1